.cirrus.yml

```diff
@@ -0,0 +1,10 @@
+version_check_task:
+  name: Test versiones go
+  env:
+    matrix:
+      - GO_VERSION: 1.19.4
```

Comment:
Por favor, no uséis versiones tan especificadas. Entended por qué usáis las versiones, y cómo hay que especificarlas.